rust/noosphere/src/ffi/context.rs

```diff
@@ -528,3 +528,28 @@ pub fn ns_sphere_file_version_get(
             .map_err(|error: InvalidNulTerminator<String>| anyhow!(error).into())
     })
 }
+
+#[ffi_export]
+/// @memberof ns_sphere_file_t
+/// Get the identity (a DID encoded as a UTF-8 string)
+/// for this ns_sphere_t.
+pub fn ns_sphere_identity_get(
```

Comment:
`ns_sphere_identity()` may be more consistently named (e.g. `ns_sphere_receipt_identity()` as similar on a `ns_sphere_receipt_t` object)

Reply:
Done!